### PR TITLE
IDEMPIERE-5416 Alt+A is used for Create+New - for usability would be good to use it also for New

### PR DESCRIPTION
**Pull request by CarlosRuiz-globalqss**
https://idempiere.atlassian.net/browse/IDEMPIERE-5416

### CODE REVIEW

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/AbstractADWindowContent.java [hunk 1]
```diff
@@ -2721,21 +2721,18 @@ public void onSaveCreate()
...
+				IADTabpanel dirtyTabpanel = (IADTabpanel) Executions.getCurrent().removeAttribute("adtabpane.saved");
+				if (dirtyTabpanel != null && dirtyTabpanel.getGridTab().isDetail()) {
+					try {
+						adTabbox.getSelectedTabpanel().getDetailPane().onNew();
+					} catch (Exception e) {
+						throw new RuntimeException(e);
+					}
+				} else {
+					onNew();
+				}
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/AbstractADWindowContent.java:2735]: why remove the `if(result)` checking ?

Comment (CarlosRuiz-globalqss) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/AbstractADWindowContent.java:2735]: result is the return of the onSave operation.
The dataSave is returning ERROR when the record has not been modified, and there is nothing to save, and consequently the onSave is returning false, so the onNew is not called in this case.  I tested it and it seems to work without problems.

Comment (hengsin) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/AbstractADWindowContent.java:2735]: Not working well for the following:
- Open Product Window
- Create new record, Enter Weeder for both Search Key and Name
- Press Alt+A to save

Comment (CarlosRuiz-globalqss) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/AbstractADWindowContent.java]: Thanks @hengsin - I added a new commit that works better.

**Review by hengsin — APPROVED** [review]
tested

**Pull request merged**